### PR TITLE
New Rule: (require | disallow)SpacesInsideImportedObjectBraces

### LINKED ISSUE #2151: Require spaces for object destructuring in imports, alignment cases for imports
What about rule for require spaces for import cases?

```
invalid:
import {foo,bar} from 'foobar';

valid:
import { foo,bar } from 'foobar';
```

and(maybe for another rule):
```
invalid:
import { foo,bar } from 'foobar';
import { foo as f,bar } from 'foobar';

valid:
import { foo, bar } from 'foobar';
import { foo as f, bar } from 'foobar';
```
and what about alignment cases?

```
invalid:
import { foo, bar } from 'foobar';
import { a, b, c, d, e } from 'alphabet';

valid:
import { foo, bar }      from 'foobar';
import { a, b, c, d, e } from 'alphabet';

```
but I'm not sure how we can detect alignment for very long import strings:

```
import { foo, bar} from 'foobar';
import { a as SomethingWithVeryLongNameA, b as SomethingWithVeryLongNameB, c as SomethingWithVeryLongNameC } from 'alphabet'
```

### PR DESCRIPTION
**Pull request by fxmaxvl**
Restrictions for spaces inside imported objects

Fixes #2151

### CODE REVIEW

**Comment by fxmaxvl** [comment]
Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
??

**Comment by fxmaxvl** [comment]
review?

**Comment by markelog** [comment]
It happens with coveralls sometimes, for example, now it all the way down...

Would you like to also update presets? At least `airbnb` is candidate.

**Comment by fxmaxvl** [comment]
ok

File: presets/airbnb.json [hunk 1]
```diff
@@ -76,5 +76,6 @@
     "validateIndentation": 2,
     "maximumLineLength": 100,
     "disallowArrayDestructuringReturn": true,
-    "requireShorthandArrowFunctions": true
+    "requireShorthandArrowFunctions": true,
+    "requireSpacesInsideImportedObjectBraces": true
```
Comment (markelog) [inline presets/airbnb.json]: It seems we need add an some `import` statement to https://github.com/jscs-dev/node-jscs/blob/fbe9bf4ad5ab3e2645affee07a5fc5ff19cea35f/test/data/options/preset/airbnb.js ?

Comment (fxmaxvl) [inline presets/airbnb.json]: but, how can i write  a correct case with import there? had trouble with it

Comment (markelog) [inline presets/airbnb.json]: Adding `import { resolve } from "path";` in the begging ?

Comment (fxmaxvl) [inline presets/airbnb.json]: Oh, tnx) I had doubts about the use of real modules for this case

Comment (markelog) [inline presets/airbnb.json]: That file will not be executed anyway, you can even add a non-existent module, will still work

**Comment by markelog** [comment]
Awesome, thank you

**Pull request closed without merging**